x/ibc/04-channel/keeper/packet_test.go

```diff
@@ -116,81 +177,97 @@ func (suite *KeeperTestSuite) TestSendPacket() {
 
 }
 
+// TestRecvPacket test RecvPacket on chainB. Since packet commitment verification will always
+// occur last (resource instensive), only tests expected to succeed and packet commitment
+// verification tests need to simulate sending a packet from chainA to chainB.
 func (suite *KeeperTestSuite) TestRecvPacket() {
-	counterparty := types.NewCounterparty(testPort1, testChannel1)
-	packetKey := host.KeyPacketCommitment(testPort2, testChannel2, 1)
-
-	var packet exported.PacketI
+	var (
+		packet exported.PacketI
+	)
 
 	testCases := []testCase{
 		{"success ordered channel", func() {
```

Comment:
Add test to fail receiving out of order on ORDERED channel

Reply:
Ohhh I remember why I didn't add this test, the seq checking happens in PacketExecuted not RecvPacket, see [pr](https://github.com/cosmos/cosmos-sdk/pull/6312#issue-426285476). Seems like it'd be good to just add this check into `RecvPacket` will comment out my test and open an issue to be addressed in a separate pr